app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
I'm in two minds about this. On the one hand it may be useful and wouldn't be too hard to implement. On the other hand, it's not functionality that's needed anywhere so I'm not convinced it should be done. Maybe if a story comes up that needs to filter by a different user role it could be implemented then?